### PR TITLE
Replaces PNGS by SVG alternatives

### PR DESCRIPTION
**Pull request by SrNetoChan**
Also, creates new Icons for layer visibility

### CODE REVIEW

**Comment by SrNetoChan** [comment]
@nirvn Can you review please?

File: src/app/qgisapp.cpp [hunk 1]
```diff
@@ -1704,7 +1704,7 @@ void QgisApp::createActions()
   connect( mActionHideAllLayers, SIGNAL( triggered() ), this, SLOT( hideAllLayers() ) );
   connect( mActionShowSelectedLayers, SIGNAL( triggered() ), this, SLOT( showSelectedLayers() ) );
   connect( mActionHideSelectedLayers, SIGNAL( triggered() ), this, SLOT( hideSelectedLayers() ) );
-  connect( mActionHideDeselectedLayers, &QAction::triggered, this, &QgisApp::hideDeselectedLayers );
```
Comment (nyalldawson) [inline src/app/qgisapp.cpp]: This shouldn't be reverted

Comment (SrNetoChan) [inline src/app/qgisapp.cpp]: Sorry, my mistake. For some reason, at a point, the Icon for hideDeselectedLayers was not showing up. I was trying to find a reason for it. Apparently, only saving the icon again made it work and then I forgot to revert this.

Thanks for the warning. It's fixed now.

**Comment by nirvn** [comment]
@SrNetoChan , great job, keep those coming 😄

**Pull request merged**